lib/credential/domain/use_cases/get_claims_use_case.dart

```diff
@@ -36,22 +36,25 @@ class GetClaimsUseCase
     // if profileNonce is zero, return all profiles credentials,
     // if profileNonce > 0 then return only credentials from that profile
     if (param.profileNonce >= GENESIS_PROFILE_NONCE) {
-      // TODO check param.did and did from profile nonce are the same or return exception
       String did = await _getCurrentEnvDidIdentifierUseCase.execute(
           param: GetCurrentEnvDidIdentifierParam(
               privateKey: param.privateKey, profileNonce: param.profileNonce));
-      return _credentialRepository
-          .getClaims(
-              filters: param.filters,
-              genesisDid: param.genesisDid,
-              privateKey: param.privateKey)
-          .then((claims) {
-        logger().i("[GetClaimsUseCase] Claims: $claims");
-        return claims;
-      }).catchError((error) {
-        logger().e("[GetClaimsUseCase] Error: $error");
-        throw error;
-      });
+      if (param.genesisDid == did) {
```

Comment:
This check is not valid in this case (the TODO was deprecated and misleading), but it raised a concern about the check and I fixed it here: https://github.com/0xPolygonID/polygonid-flutter-sdk/pull/315
Can you revert this file in this PR?

Reply:
@Plumillon got it, have reverted the change and local unused variable did to make analyzer happy.

Reply:
@Plumillon hope you are doing fine, can you PTAL?

Reply:
Hey @AyushBherwani1998!
I'm not a maintainer anymore, please refer to @rauljareno and @plusema86 :)